charts/thoras/templates/monitor/secret.yaml

```
@@ -2,7 +2,7 @@
 ---
 apiVersion: v1
 data:
-  webhookUrl: {{ .Values.slackWebhookUrl | b64enc }}
+  webhookUrl: "{{ .Values.slackWebhookUrl | b64enc }}"
```

[dust] Could also do:
```suggestion
  webhookUrl: {{ .Values.slackWebhookUrl | b64enc | quote }}
```